### PR TITLE
proofreading, copy-editing

### PR DESCRIPTION
**Pull request by csymons-suse**
getting started

### CODE REVIEW

File: doc/source_tech_preview/deployment/overview.rst [hunk 1]
```diff
@@ -20,13 +21,13 @@ SUSE Enterprise Storage.
      default_fontsize = 11;
      deployer [label="Setup deployer"]
      ses_integration [label="SES Integration"]
-     configure [label="Configure\n Cloud"]
+     configure_soc [label="Configure\n Cloud"]
```
Comment (jgu17) [inline doc/source_tech_preview/deployment/overview.rst]: we already moved the rest from configure_soc to configure in the section below on the branch.

File: doc/source_tech_preview/index.rst [hunk 2]
```diff
@@ -1,12 +1,9 @@
-Welcome to socok8s repo
+Welcome to SUSE OpenStack Cloud on Kubernetes
```
Comment (jgu17) [inline doc/source_tech_preview/index.rst]: ->SUSE Containerized OpenStack. That's the product that T.R. wants to use.

File: doc/source_tech_preview/index.rst [hunk 3]
```diff
@@ -1,12 +1,9 @@
...
 =======================
 
-This project automates the SUSE Openstack Cloud provisioning and lifecycle
-management on SUSE Container as a Service Platform and SUSE Enterprise 
-Storage (SES) via Airship and a series of shell scripts and Ansible playbooks.
-
-Please check on the left sidebar for content. If you are interested to install
-this software, follow the deployment guide.
+The socok8s project automates SUSE OpenStack Cloud provisioning and lifecycle
```
Comment (jgu17) [inline doc/source_tech_preview/index.rst]: SUSE Containerized OpenStack...

File: doc/source_tech_preview/index.rst (hunk 2 above)
Comment (csymons-suse) [reply doc/source_tech_preview/index.rst]: nice. better.

**Pull request merged**